packages/core/core/src/Transformation.js

```diff
@@ -102,7 +102,10 @@ export default class Transformation {
     for (let {request, result} of this.configRequests) {
       let plugin =
         request.plugin != null &&
-        (await this.parcelConfig.loadPlugin(request.plugin));
+        (await this.parcelConfig.loadPlugin({
+          packageName: request.plugin,
+          resolveFrom: this.parcelConfig.filePath,
```

Comment:
Hmm, we should probably store where the plugin came from originally as part of the request